update-dependencies.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -o nounset -o errexit
+
+MY_MODULES=$(git submodule status | awk '{print $2}')
+
+for m in $MY_MODULES; do
+    cd $m
+
+    echo $m
+
+    git pull
+
+    cd ..
+done
+
+echo "Pulled updated modules. Now commit the changes."
```

Comment:
This is unclear to me. Why would you need to commit after `pull`?

Reply:
It's just the way submodules work. Whenever you update a submodule (e.g., through `git pull` or by checking out a specific commit), and you want the change reflected in the parent git repo (emirge in our case), such that a `git clone` of emirge or a `git pull` in emirge also pull the changed submodule versions, you need to commit (and push) the main repo after the modification of the submodules. 

In other words, the submodules in emirge are currently at the version I originally imported them a few weeks ago, since they haven't been updated in emirge until now. This script intends to make this update process a bit easier.

Reply:
OIC, makes sense.

```suggestion
echo "Pulled updated modules. Now commit the submodule changes to emirge."
```
(I wonder if we should remind them here to create a branch.)